### PR TITLE
VK-1 create index page and add delete/patch method to currency

### PR DESCRIPTION
**Pull request by MrRinsvind**

### CODE REVIEW

File: models/schema.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  name: {
+    type: String,
+    required: [
+      true, 'Укажите название валюты'
+    ],
+    unique: true,
+    index: true,
+  },
+  courseBuy: {
+    type: Number
```
Comment (radpoznyakov) [inline models/schema.js]: should be type string

File: models/schema.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  courseSail: {
+    type: Number
```
Comment (radpoznyakov) [inline models/schema.js]: should be type string

File: models/schema.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  description: {
+    type: String,
+    trim: true,
+    maxLength: 300,
+    default: null,
+  },
+  updatedAt: {
```
Comment (radpoznyakov) [inline models/schema.js]: why we need these types (updatedAt, createdAt)?

File: models/schema.js (hunk 1 above)
Comment (MrRinsvind) [reply models/schema.js]: managed

**Pull request merged**